pkg/asset/installconfig/openstack/validation/cloudinfo.go

```diff
@@ -197,8 +199,12 @@ func (ci *CloudInfo) getSubnet(subnetID string) (*subnets.Subnet, error) {
 }
 
 func isNotFoundError(err error) bool {
```

Comment:
I wished we had taken the chance to move the `isNotFoundError()` function next to `isUnauthorized()`.